ansible_collections/arista/avd/roles/eos_validate_state/README.md

```diff
@@ -94,7 +94,7 @@ Figure 1 below provides a visualization of the role's inputs, outputs, and tasks
   - VerifyReachability: Validate DPS reachability between devices.
 
 - AvdTestLLDPTopology
-  - VerifyLLDPNeighbors: Validate LLDP topology.
+  - VerifyLLDPNeighbors: Validate LLDP topology. To skip tests for specific Ethernet interfaces, set validate_state to False.
```

Comment:
This update does not make sense in `eos_validate_state`, since the mentioned input is deeply nested under connected endpoints. I think your update to the schema is enough. @gmuloc?

Reply:
Hmm I guess it would make sense to track somewhere which tests can be infuenced by the knob in the structured config and where to set them, Not sure whats the best way forward. Right now people cannot guess from this page that it can be disabled. Maybe another wording?

Also @MaheshGSLAB you have updated eos_designs schema but I think it would be worth to update also eos_cli_config_gen as the key shows up in :

```
../schema/schema_fragments/port_channel_interfaces.schema.yml:        validate_state:
../schema/schema_fragments/port_channel_interfaces.schema.yml:          description: Set to false to disable interface validation by the `eos_validate_state` role.
../schema/schema_fragments/ethernet_interfaces.schema.yml:        validate_state:
../schema/schema_fragments/ethernet_interfaces.schema.yml:          description: Set to false to disable interface validation by the `eos_validate_state` role.
```

@ClausHolbechArista would it be ok to make some section in this doc to list which variables can impact validate state (with full path) and then in the test documentation state that this (these) key(s) impacts the test?